src/ScottPlot5/ScottPlot5/Control/Interaction.cs

```diff
@@ -51,6 +60,7 @@ public void Disable()
     public void Enable()
     {
         Actions = ActionsWhenDisabled;
```

Comment:
I would suggest to change `Disable()` to only use `ActionsWhenDisabled` if it was disabled:
```csharp

public void Enable()
{
    if (Disabled)
        Actions = ActionsWhenDisabled;
    Disabled = false;
}
```